tex/generic/pgf/systemlayer/pgfsys-dvips.def

```diff
@@ -161,8 +170,13 @@
     /.setstrokeconstantalpha where
       {pop 1 .pgfsetfillopacityalpha 1 .pgfsetstrokeopacityalpha} if
   }%
+<<<<<<< HEAD
+  \ht#1=0pt%
+  \dp#1=0pt%
+=======
   \ht#1=0pt
   \dp#1=0pt
+>>>>>>> 972751ed77acd7187b40d7b4da1c9d617e7abda0
```

Comment:
Here seems to be something left behind ...
```suggestion
  \ht#1=0pt
  \dp#1=0pt
```

Reply:
I fixed it, hopefully. Henry asked me to merge this onto master. Is `git rebase master` the correct command?